tests/registration_test.py

```diff
@@ -125,3 +125,71 @@ def test_clear(self):
         g_twos_array = ar_array + aar_array
         self.assertTrue((twos_array.to_ndarray() == 
                                        g_twos_array.to_ndarray()).all())
+
+    def test_register_info(self):
+        '''
+        Tests the following:
+
+        1. info() with an empty symbol table returns 'the symbol table is empty' regardless of arguments
+        2. info(ak.RegisteredSymbols) when no objects are registered returns 'the registry is empty'
+        3. The registered field is set to false for objects that have not been registered
+        4. The registered field is set to true for objects that have been registered
+        5. info(ak.AllSymbols) contains both registered and non-registered objects
+        6. info(ak.RegisteredSymbols) only contains registered objects
+        '''
+        # Cleanup symbol table from previous tests
+        cleanup()
```

Comment:
RE: cleanup() -> Nice catch here, the previous tests leave behind some state since the server & SymTab are shared across the tests.